spec/replication/util/miq_pglogical_spec.rb

```diff
@@ -158,6 +158,12 @@
       expect(described_class).to receive(:refresh_excludes).with(YAML.safe_load(tables))
       described_class.save_remote_region(tables)
     end
+
+    it "does not updates list of tables to be excluded from replication if passed paramere is empty" do
```

Comment:
Is it okay to have an empty exclusion_list?  I think there are some tables we are counting on never replicating.

Reply:
Yeah, we have [this](https://github.com/ManageIQ/manageiq/blob/master/lib/miq_pglogical.rb#L10) but it's not being used very thoroughly for application-side tables (but rather for AR and other system-type tables).

Reply:
There are probably lots of tables that would cause problems if they were replicated, but we have never tried to enumerate them in any serious way.

Reply:
Also, this has some typos `s/updates/update/` `s/paramere/parameter/`